src/components/Sidebar/index.jsx

```diff
@@ -1,5 +1,5 @@
 const Sidebar = () => {
-  return <div></div>;
+  return <p>I'm a sidebar</p>;
```

Comment:
Same:
https://github.com/mim24sd/music-app/pull/3/files#r1047538311

Reply:
➕ Should be identical to design? Please add other items as well.
➕ Semantic html maybe?